### ISSUE TITLE
TwiML - Task verb is missing

### ISSUE THREAD

**Issue by mosampaio** [twilio/twilio-python#283]
**Version:** next-gen
**API Subdomain:** twiml
### Code Snippet

``` python
resp = twiml.Response()
    with resp.enqueue(None, workflowSid=workflow_sid) as e:
        e.task('{"account_number":"12345abcdef"}')
```
### Exception / Log

```
AttributeError: 'Enqueue' object has no attribute 'task'
```
### Feature Request

In the TwiML Enqueue Verb, it should be possible to append a `Task`.

In the previous version, it [exists](https://github.com/twilio/twilio-python/blob/master/twilio/twiml.py#L537), but I don't see it in the next-gen.

**Comment by efossier** [comment]
@mosampaio Thanks for bringing this to our attention, looks to be an oversight on our part. We'll get this fixed soon.

**Comment by jmctwilio** [comment]
https://github.com/twilio/twilio-python/pull/336

**Comment by jmctwilio** [comment]
Fixed by Release 6.1.1

**Comment by fehrenbacher** [comment]
Several examples [here](https://www.twilio.com/docs/api/taskrouter/twiml-integration) are incomplete and reference this issue. An update would be very helpful!
